lab_1_classify_by_unigrams/main.py

```diff
@@ -28,6 +42,11 @@ def create_language_profile(language: str, text: str) -> dict[str, str | dict[st
     :param text: a text
     :return: a dictionary with two keys – name, freq
     """
+    if not isinstance(language, str) or not isinstance(text, str):
+        return None
+    dict_of_freq = calculate_frequencies(tokenize(text))
+    lang_prof = {'name': language, 'freq': dict_of_freq}
```

Comment:
do not create the variable - return as is

Reply:
done